### PR TITLE
Docs redisdbindexer crud

### PR DESCRIPTION
**Pull request by florian-hoenicke**

### CODE REVIEW

File: indexers/keyvalue/RedisDBIndexer/__init__.py [hunk 1]
```diff
@@ -51,13 +52,15 @@ def query(self, key: int, *args, **kwargs) -> Optional[bytes]:
...
         return results[0]['values']
 
     def add(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
-        """Add a JSON-friendly object to the indexer
-        :param keys: keys to be added
-        :param values: values to be added
+        """Add JSON-friendly serialized documents to the index.
+
+        :param keys: document ids
+        :param values: JSON-friendly serialized documents
```
Comment (JoanFM) [inline indexers/keyvalue/RedisDBIndexer/__init__.py:63]: is correct with the type definition?

File: indexers/keyvalue/RedisDBIndexer/__init__.py [hunk 2]
```diff
@@ -66,7 +69,10 @@ def add(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
                 redis_handler.set(k['_id'], k['values'])
 
     def update(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
-        """updates the keys if they exist
+        """Update JSON-friendly serialized documents on the index.
```
Comment (JoanFM) [inline indexers/keyvalue/RedisDBIndexer/__init__.py:72]: same herr

File: indexers/keyvalue/RedisDBIndexer/__init__.py (hunk 1 above)
Comment (florian-hoenicke) [reply indexers/keyvalue/RedisDBIndexer/__init__.py:63]: What I mean is that the document is serialized, i.e., in byte form. Also it is required to be in a json friendly format after deserialization.

Comment (JoanFM) [reply indexers/keyvalue/RedisDBIndexer/__init__.py:63]: Okey, good thanks

File: indexers/keyvalue/RedisDBIndexer/__init__.py [hunk 3]
```diff
@@ -33,10 +33,11 @@ def get_query_handler(self):
         except redis.exceptions.ConnectionError as r_con_error:
             self.logger.error('Redis connection error: ', r_con_error)
 
+    # TODO unify result interface
     def query(self, key: int, *args, **kwargs) -> Optional[bytes]:
-        """Find the protobuf chunk/doc using id
+        """Find the protobuf documents via id.
```
Comment (nan-wang) [inline indexers/keyvalue/RedisDBIndexer/__init__.py:38]:
```suggestion
        """Find the protobuf document via id.
```

**Review by nan-wang — CHANGES_REQUESTED** [review]
LGTM👍

**Comment by florian-hoenicke** [comment]
Can it be merged? @nan-wang

**Review by nan-wang — APPROVED** [review]

**Pull request merged**